tests/input/input_diffuse.cpp

```diff
@@ -90,8 +88,7 @@ TEST_CASE("Input reading using GetKw for an input file for a diffuse environment
   REQUIRE(units == parsedInput.units());
   REQUIRE(CODATAyear == parsedInput.CODATAyear());
   REQUIRE(type == parsedInput.cavityType());
-  REQUIRE(patchLevel == parsedInput.cavityParams().patchLevel);
-  REQUIRE(coarsity == Approx(parsedInput.cavityParams().coarsity));
+  REQUIRE(area == Approx(parsedInput.cavityArea()));
```

Comment:
You can get the area from `cavityParams().area`, instead of adding a new getter to the input object.